packages/special-pages/pages/duckplayer/src/js/index.js

```diff
@@ -301,18 +302,20 @@ const Comms = {
      * Validates that the input string is a valid video id.
      * If so, returns the video id otherwise returns false.
      * @param {string} input
-     * @returns {(string|false)}
+     * @returns {(string|null)}
      */
     validateVideoId: (input) => {
```

Comment:
This should probably be renamed to sanitise rather than validate if we're intending to manipulate.